fjord/feedback/templates/feedback/generic_feedback.html

```diff
@@ -27,7 +27,7 @@
         <div class="header-content">
           <div class="left-side">
             <div id="back-button-container" tabindex=0>
-              <svg id="back-button" viewBox="0 0 55 100">
+              <svg id="back-button" viewBox="0 -10 50 125">
```

Comment:
This affects the portion of the svg image that gets displayed. You don't really want to change these numbers since they're already the edges of the image. Instead you want to move the position of the image in relation to the header background. That can be done solely with CSS.